targetcli/ui_backstore.py

```diff
@@ -423,17 +423,17 @@ def _create_file(self, filename, size, sparse=True):
             raise ExecutionError("Could not open %s" % filename)
         try:
             if sparse:
+                os.ftruncate(f.fileno(), size)
+            else:
+                self.shell.log.info("Writing %d bytes" % size)
                 try:
                     os.posix_fallocate(f.fileno(), 0, size)
                 except AttributeError:
                     # Prior to version 3.3, Python does not provide fallocate
```

Comment:
@maurizio-lombardi nit: just move the above comment line before call to posix_fallocate()
Rest all looks good.